collect_events.py

```diff
@@ -1,24 +1,71 @@
 import json
+import logging
 import os
+import time
 
+import jwt
 import requests
 
-# Twosense API access token. Get it here:
-# https://twosense.readme.io/reference/authentication#2-obtain-an-access-token
-API_TOKEN = os.getenv("TWOSENSE_API_TOKEN")
+logging.basicConfig(
+    filename="app.log",
+    filemode="a",
+    format="%(asctime)s [%(levelname)s] - %(message)s",
+    level=logging.INFO,
+)
+
+CLIENT_ID = os.getenv("TWOSENSE_API_CLIENT_ID")
+CLIENT_SECRET = os.getenv("TWOSENSE_API_CLIENT_SECRET")

```

Comment:
should we add the possibility to get the credentials from a file as well?

Get them from the env variables, if not there from a configuration file, if not there then they don't eixst

Reply:
I think for a sample script this is good enough. The developers will adapt it for their specific use case anyway.